spec/requests/api/v1/dynamic_field_groups_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'rails_helper'
 
 RSpec.describe 'Dynamic Field Groups Requests', type: :request do
+  let(:field_export_profile) { FactoryBot.create(:field_export_profile) }
+
+  before { field_export_profile }
```

Comment:
Can you use `let!` in the let statement above instead of having this before block?

Reply:
Whoa, I totally forgot about `let!`.

Will do.